### DISCUSSION TITLE
How to make node legends for paga graph

### DISCUSSION THREAD

**Discussion by Chengwei94** [theislab/scvelo#819] [Q&A] [answered]
Hi there, 

this is not really a bug but just a question. How do I make the node legend appear when drawing paga. Right now, it is just the clustering labels that appear. Is there a easy way to make this happen? 

Heres my code :

scv.pl.paga(retina1, basis='umap', size=50, alpha=0.4, color = "type", save = "paga")   

![image](https://user-images.githubusercontent.com/61018420/152479614-e9c1a160-3e9a-439f-b43f-518391dc44a8.png)

**Comment by WeilerP** [comment, marked as answer]
@Chengwei94, clusters used to calculate PAGA are also used for annotation.

```python
scv.tl.paga(adata, groups='cell_type')
scv.pl.paga(adata, basis='umap')
```

will use the values of `adata.obs['cell_type']` in the legend.

**On a general note:** If this is not a bug, please do not open a bug issue. The discussions are the right place to ask these kind of questions. For general inquiries regarding the actual code base, you may open a normal issue (without any labels).
